pages/linux/zathura.md

```diff
@@ -0,0 +1,25 @@
+# zathura
+
+> A vim-like modal document viewer, with an integrated command line.
+> Make sure a backend is installed (poppler, PostScript or DjVu).
+> More information: <https://pwmt.org/projects/zathura/>.
+
+- Navigate left/up/down/right:
+
+`<H|J|K|L> or arrow keys`
+
+- Rotate:
+
+`r`
+
+- Invert Colors:
+
+`Ctrl + R`
+
+- Search for text:
+
+`/`
+
+- Create/delete/list bookmarks:
+
+`:{{bmark|bdelete|blist}} {{bookmark_name}}`
```

Comment:
You can list a specific bookmark?

Reply:
`:blist` is used without arguments, my bad. Is there a way yo include it in this entry (I don't think it rates it's own)?

Reply:
I think it's probably best as a separate example.